### PR TITLE
fix: remove deprecated {{title}} ast transform

### PR DESCRIPTION
**Pull request by knownasilya**
BREAKING CHANGE: removes deprecated `{{title}}` ast transform, rename to `{{page-title}}`

### CODE REVIEW

File: addon/services/page-title-list.js [hunk 1]
```diff
@@ -189,12 +190,12 @@ export default Service.extend({
...
       );
-    }
+    },
   }),
 
   scheduleTitleUpdate() {
-    let router = getOwner(this).lookup('router:main');
-    let { activeTransition } = router._routerMicrolib;
+    // eslint-disable-next-line ember/no-private-routing-service
+    let { activeTransition } = this.router._router._routerMicrolib;
```
Comment (knownasilya) [inline addon/services/page-title-list.js:198]: @rwjblue any way to get the `activeTransition` without private APIs?

Comment (rwjblue) [reply addon/services/page-title-list.js:198]: Should be fixed with a rebase now

File: addon/services/page-title-list.js [hunk 2]
```diff
@@ -189,12 +190,12 @@ export default Service.extend({
...
         groups.reduce((E, group) => E.concat(group), [])
       );
-    }
+    },
   }),
 
   scheduleTitleUpdate() {
-    let router = getOwner(this).lookup('router:main');
-    let { activeTransition } = router._routerMicrolib;
+    // eslint-disable-next-line ember/no-private-routing-service
```
Comment (rwjblue) [inline addon/services/page-title-list.js:197]: Can probably remove this in a rebase now...

**Review by rwjblue — APPROVED** [review]
Awesome, great job!

**Pull request merged**